### PR TITLE
*: improve bazel config

### PR DESCRIPTION
**Pull request by hawkingrei**
### What problem does this PR solve?

Issue Number: close #xxx

Problem Summary:

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

**Review by lance6716 — APPROVED** [review]

File: store/BUILD.bazel [hunk 1]
```diff
@@ -25,6 +25,7 @@ go_test(
     ],
     embed = [":store"],
     flaky = True,
+    shard_count = 23,
```
Comment (Defined2014) [inline store/BUILD.bazel:28]: If we add a new test in store, should we increase the `shard_count` here?

Comment (hawkingrei) [reply store/BUILD.bazel:28]: Normally, you don't need to worry about this. Unless you write a flaky test here, we will increase it when this package's test is slow.

**Pull request merged**